andela/travel_tool_back
# reconfigure eslint and fix errors

#### What does this PR do?
Introduces eslint rules that make the codebase uniform and readable

#### Description of Task to be completed?
enable the following eslint rules:

- end-of-line
- no-unused variables
- lines not exceeding maximum of 100
- no trailling spaces
- spaces after opening and closing braces

#### How should this be manually tested?
- After pulling latest changes from develop, run ```yarn install```
- When a trailling or unused variable is introduced, esling should prompt an error about that

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[161323203](https://www.pivotaltracker.com/story/show/161323203)

#### Screenshots (if appropriate)

#### Questions:
